src/utils.js

```diff
@@ -0,0 +1,4 @@
+export function isObject(value) {
+  var type = typeof value;
+  return !!value && (type === 'object' || type === 'function');
```

Comment:
This is more naïve than Lodash impl. For example this will return `true` for arrays.